intuita-webview/src/main/TreeView/index.tsx

```diff
@@ -14,6 +16,28 @@ type Props = {
 	node: TreeNode;
 };
 
+const getIcon = (iconName?: string): ReactElement => {
```

Comment:
what does it mean an `iconName` is undefined? If it can be "empty", maybe we should switch to `string | null` type.

Reply:
Is this what you mean? https://github.com/intuita-inc/intuita-vscode-extension/pull/237/commits/51535af24d735a09fc6b95cc06586e54ffeecc31